Cognigy/WebchatWidget
# feature/16692 add logic to use QuickReply title as postback

Description:

This PR adds a functionality that clicked QuickReply buttons return the title instead of the payload in case the useQuickReplyTitleAsPostback option is set to true 

Success criteria:

- QuickReplies return the title as postback if the flag is set

How to test (steps):

1. Embed the webchat with the setting useQuickReplyTitleAsPostback enabled
2. Create a flow with a say node with a quick reply with a title other than the postback
3. Create another say node with a Messenger channel QuickReply with a title different than the payload
4. Check the response in another saynode sending the response {{ci.text}} it should return the title instead of the payload

Additional considerations (describe):

  - Examples: adding, renaming, removing a field or changing the format or constraints of a field

Security implications (describe):
